### PR TITLE
Update react checklists

### PR DESCRIPTION
**Pull request by SpaNb4**

### CODE REVIEW

**Comment by SpaNb4** [comment]
@ragingyngvarr, ревьювни плиз

**Comment by SpaNb4** [comment]
@kravaring need your review

File: active-tasks/react_components.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      "max": -50
+    },
+    {
+      "type": "penalty",
+      "text": "React hooks are used to get access to either state or to the component lifecycle",
+      "max": -70
+    },
+    {
+      "type": "penalty",
+      "text": "Presence of code smells (God-object, chunks of duplicate code), commented code sections",
```
Comment (kravaring) [inline active-tasks/react_components.json:72]: I guess -10 per each

File: active-tasks/react_nextJs.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      "max": -20
+    },
+    {
+      "type": "penalty",
+      "text": "Direct DOM manipulations inside the React components",
+      "max": -50
+    },
+    {
+      "type": "penalty",
+      "text": "Presence of code-smells (God-object, chunks of duplicate code), commented code sections",
```
Comment (kravaring) [inline active-tasks/react_nextJs.json:57]: per each

File: active-tasks/react_redux.json [hunk 3]
```diff
@@ -1,90 +1,86 @@
...
+      "max": -20
+    },
+    {
+      "type": "penalty",
+      "text": "Direct DOM manipulations inside the React components",
+      "max": -50
+    },
+    {
+      "type": "penalty",
+      "text": "Presence of code-smells (God-object, chunks of duplicate code), commented code sections",
```
Comment (kravaring) [inline active-tasks/react_redux.json:67]: per each

File: active-tasks/react_redux.json [hunk 4]
```diff
@@ -1,90 +1,86 @@
...
+      "max": 20
+    },
+    {
+      "type": "penalty",
+      "text": "TypeScript isn't used",
+      "max": -95
+    },
+    {
+      "type": "penalty",
+      "text": "Usage of any",
```
Comment (kravaring) [inline active-tasks/react_redux.json:52]: I think that here also should be "per each"

**Pull request merged**